docker/Dockerfile

```diff
@@ -19,11 +19,12 @@ RUN echo "## Start building" \
     && echo "## Done"
 
 RUN echo "## Get EMSDK" \
-    && git clone https://github.com/emscripten-core/emsdk.git ${EMSDK} \
+    && git clone --depth=1 https://github.com/emscripten-core/emsdk.git ${EMSDK} \
```

Comment:
This whole step seems a little odd given that this Dockerfile now lives in the emsdk repo itself.  I've been working on a PR to remove this step and just have the Dockerfile operate of the current checkout that contains it.  I'm still up for landing this change first as-is though.

Reply:
Great! Copying the current checkout to the Dockerfile is indeed a better way to handle this.

Reply:
Commit https://github.com/emscripten-core/emsdk/pull/571/commits/118f0919659c43ab7c282c95917ccdf8bf952fc8 operates on the current checkout directory within the Dockerfile. This ensures that any changes in emsdk itself or in the Docker test script (i.e. `docker/test_dockerimage.sh`) are tested during pull requests.